operators/src/processing/column_range_filter.rs

```diff
@@ -0,0 +1,281 @@
+use crate::engine::{
+    ExecutionContext, InitializedOperator, InitializedOperatorImpl, InitializedVectorOperator,
+    Operator, QueryContext, QueryProcessor, QueryRectangle, TypedVectorQueryProcessor,
+    VectorOperator, VectorQueryProcessor, VectorResultDescriptor,
+};
+use crate::error;
+use crate::util::input::StringOrNumber;
+use crate::util::Result;
+use failure::_core::marker::PhantomData;
+use failure::_core::ops::RangeInclusive;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::collections::FeatureCollection;
+use geoengine_datatypes::primitives::{FeatureDataType, FeatureDataValue, Geometry};
+use geoengine_datatypes::util::arrow::ArrowTyped;
+use serde::{Deserialize, Serialize};
+use snafu::ensure;
+use std::convert::TryInto;
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct ColumnRangeFilterParams {
+    pub column: String,
```

Comment:
column_name? we need some convention in how to name such parameters.

Reply:
btw is it columns or attributes now for featurecollections? in geojson/wfs/ogr they are called properties. In mapping we called them attributes i think

Reply:
:shrug: if guess we can use the table and column naming… but you are right if it we should decide on `column_name` or just `column`… what is better for the user input?

Reply:
where do we use the term "table" in our project? We have to really think about going against the convention of the major tools and standards of calling attributes of features properties instead of columns.

About user input: maybe we should prefer more concise terms. The parameters need to have documentation anyway and we can specify the semantics more precisely there.

Reply:
So you vote for attributes?

Reply:
_Attributes_ sound most natural for me, however _properties_ is more compliant to other software and standard **and** reflects the simple feature model more than _columns_. There are no columns in a simple feature collection as each feature can have a separate set of properties. Of course columns do reflect **our** model more.

Reply:
-> separate Task